apps/api/src/app/workflows-v2/usecases/sync-to-environment/sync-to-environment.usecase.ts

```diff
@@ -144,7 +144,10 @@ export class SyncToEnvironmentUseCase {
     });
   }
 
-  private buildStepCreateOrUpdateDto(step: StepDataDto, existingInternalId?: string): StepUpdateDto | StepCreateDto {
+  private buildStepCreateOrUpdateDto(
+    step: StepResponseDto,
+    existingInternalId?: string
+  ): StepUpdateDto | StepCreateDto {
```

Comment:
it's weird now to see that some functions require `StepResponseDto`

Reply:
exactly why I renamed it.